sdk/servicebus/service-bus/samples/typescript/src/interactiveLogin.ts

```diff
@@ -13,10 +13,15 @@
 import { ServiceBusClient } from "@azure/service-bus";
 import { interactiveLogin } from "@azure/ms-rest-nodeauth";
 
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
 // Define Service Bus Endpoint here
-const serviceBusEndpoint = ""; // <your-servicebus-namespace>.servicebus.windows.net
+const serviceBusEndpoint =
+  process.env.SERVICE_BUS_ENDPOINT || "<your-servicebus-namespace>.servicebus.windows.net";
```

Comment:
Not a huge deal but your other samples have just fell back to "" (and I assume they should know that the sample.env would contain the actual explanation of the setting and it's format).

Reply:
I'm torn. On the one hand I think it's nice to have the basic documentation of structure for the expected value in the string literal here. Since we aren't saying "you have to use sample.env" in the readmes and are instead presenting it as an option, I think I want to just make the rest of them look like these instead.